components/speedreader/speedreader_rewriter_service.cc

```diff
@@ -35,16 +37,30 @@ namespace {
 constexpr const char kSpeedreaderStylesheet[] = "speedreader-stylesheet";
 
 std::string WrapStylesheetWithCSP(const std::string& stylesheet) {
```

Comment:
Which stylesheet is this modifying?

Have you tried loading custom fonts via CSS using the `@import url('')` method?

Reply:
speedreader-desktop.css

Reply:
@import added instead of link rel